gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -4,15 +4,23 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-An API gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services
-to help you separate orchestration from implementation concerns. You can then leverage the governance capabilities
-of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+An API gateway is an intermediary between a client application and a set of microservices or backend APIs. Responsibilities include request routing, rate limiting, caching, authentication, authorization, security, monitoring, and logging.
+
+A gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns.
+
+Features of an API gateway include:
+
+* Request routing. A gateway routes requests to the appropriate microservice based on the request path or other conditions.
```

Comment:
Suggest formatting the bullet list like this to make it more scannable:

```
* Request routing
+
A gateway....
```

Reply:
Fixed